web/e2e/page-objects/entities/entities.ts

```diff
@@ -58,14 +58,46 @@ export class EntityPage extends AppPage {
     return element(by.css('mdl-textfield[label=Version] input'));
   }
 
+  async setEntityVersion(input: string) {
+    let inputField = this.entityVersion;
+    await inputField.clear();
+    return await inputField.sendKeys(input);
+  }
+
   get entityDescription() {
     return element(by.css('mdl-textfield[label=Description] input'));
   }
 
+  async setEntityDescription(input: string) {
+    let inputField = this.entityDescription;
+    await inputField.clear();
+    return await inputField.sendKeys(input);
+  }
+
+  async clearInputField(element) {
+    await element.click();
+    for (let i = 0; i < 50; i++) {
+      await browser.actions().sendKeys(Key.ARROW_LEFT).perform();
+      await browser.actions().sendKeys(Key.DELETE).perform();
+    }
```

Comment:
Did we try select all and delete, instead of using for loop?
This might get flaky if at some point the cursor is at the beginning or end of the text field.

Reply:
You can try based on os ctrl + a or command + a

Reply:
Yes, we did try to select and remove instead of using for loop. However, it doesn't work on MAC probably because there is no ctrl button there. Thanks for the good advice we need to make sure the cursor at the end or at the beginning of the string. I will change this method.

Reply:
Agree, if we have a string with 70 characters, this won't work

Reply:
Increased to 100. I tried a few different approaches but they are not reliable.